experiments/interpretation/dutchf3_patch/local/test.py

```diff
@@ -414,17 +425,35 @@ def test(*options, cfg=None, debug=False):
     score, class_iou = running_metrics_overall.get_scores()
 
     logger.info("--------------- FINAL RESULTS -----------------")
-    logger.info(f'Pixel Acc: {score["Pixel Acc: "]:.3f}')
-    for cdx, class_name in enumerate(_CLASS_NAMES):
-        logger.info(f'     {class_name}_accuracy {score["Class Accuracy: "][cdx]:.3f}')
-    logger.info(f'Mean Class Acc: {score["Mean Class Acc: "]:.3f}')
-    logger.info(f'Freq Weighted IoU: {score["Freq Weighted IoU: "]:.3f}')
-    logger.info(f'Mean IoU: {score["Mean IoU: "]:0.3f}')
+    logger.info(f'Pixel Acc: {score["Pixel Acc: "]:.4f}')
+
+    if debug:
+        for cdx in range(n_classes):
+            logger.info(f'  Class_{cdx}_accuracy {score["Class Accuracy: "][cdx]:.3f}')
+    else:
+        for cdx, class_name in enumerate(_CLASS_NAMES):
+            logger.info(f'     {class_name}_accuracy {score["Class Accuracy: "][cdx]:.4f}')
+
```

Comment:
this if debug statement and the one before are almost identical.. hard to see here without the whole file but is it a duplicate? can we have them all under the same if statement if they're not?

Reply:
one is per split and the other is for final results, agree that they look similar - without refactoring test.py can't do much about it... flag refactoring test.py as a separate github issue?